src/storage/src/hummock/compactor_tests.rs

```diff
@@ -449,7 +449,7 @@ mod tests {
 
         let existing_table_id = 2;
         let kv_count = 1001;
-        let base_epoch = Epoch::now();
+        let base_epoch = Epoch(0);
```

Comment:
I am not convinced that using `Epoch::now()` makes the unit test unstable. We'd better investigate on the root cause of instability.

Reply:
fix it , the reason is that `scan` only return the user key, its not Include the `epoch`